spark-jobs/src/main/scala/filodb.downsampler/index/DSIndexJob.scala

```diff
@@ -4,18 +4,20 @@ import scala.concurrent.Await
 
 import com.typesafe.config.Config
 import com.typesafe.scalalogging.StrictLogging
-import monix.execution.{Scheduler, UncaughtExceptionReporter}
+import monix.execution.Scheduler
 
 import filodb.cassandra.FiloSessionProvider
 import filodb.cassandra.columnstore.CassandraColumnStore
+import filodb.core.{DatasetRef, Instance}
+import filodb.core.metadata.Schemas
 import filodb.core.store.PartKeyRecord
-import filodb.downsampler.BatchDownsampler._
 import filodb.downsampler.DownsamplerSettings
 import filodb.downsampler.DownsamplerSettings.rawDatasetIngestionConfig
 import filodb.memory.format.UnsafeUtils
 
-object DSIndexJob extends StrictLogging {
+object DSIndexJob extends StrictLogging with Instance {
```

Comment:
Dont understand why `with Instance` is needed

Reply:
for creating session provider - line no: 46. In earlier code session provider defined in BatchDownsampler object was initialized.